src/containers/LoginForm/LoginForm.tsx

```diff
@@ -15,13 +9,13 @@ import {
   Title,
   Wrapper,
 } from "./LoginForm.styled";
+const baseApiUrl = import.meta.env.VITE_BASE_URL?.toString();
 
 export function LoginForm() {
   const signinWithGitHub = async () => {
     try {
-      const response = await api.get("/api/login/github");
-      console.log(response);
-    } catch (error) {
+      window.location.href = `${baseApiUrl}api/login/github`;
+    } catch (error: any) {
```

Comment:
do we need f() can we put just in comp link and remove window.location.href, BTW is there any reason to use try catch?